Makefile

```diff
@@ -0,0 +1,144 @@
+# From https://raw.githubusercontent.com/vincentbernat/hellogopher/1cf9ebeeaea41d2e159e196f7d176d6b12ff0eb2/Makefile
```

Comment:
This Makefile is inspired by a popular template for Go projects. I didn't invent all of this. The only modifications I brought are:

* Installation of additional tools:
  * Gengo
  * Addlicense
  * Mockery